extension/chrome/settings/index.ts

```diff
@@ -425,18 +425,16 @@ View.run(class SettingsView extends View {
       const created = new Date(prv.created);
       const date = Str.monthName(created.getMonth()) + ' ' + created.getDate() + ', ' + created.getFullYear();
       const escapedFp = Xss.escape(ki.fingerprint);
-      let escapedPrimaryOrRm = '';
-      if (ki.primary) {
-        escapedPrimaryOrRm = '(primary)';
-      } else if (canRemoveKey) {
-        escapedPrimaryOrRm = `(<a href="#" class="action_remove_key" data-test="action-remove-key" fingerprint="${escapedFp}">remove</a>)`;
+      let removeKeyBtn = '';
+      if (canRemoveKey) {
```

Comment:
Careful - `canRemoveKey` here means it's not forbidden by any policy, but that doesn't mean we should let users remove their very last key. Even if they normally are allowed to remove keys, they should not be allowed to remove the key if they only have one loaded - then the extension would become unusable.